deploy/upstream/05_role.yaml

```diff
@@ -13,12 +13,21 @@ rules:
   - delete
   - update
   - list
+- apiGroups:
```

Comment:
You cannot just copy paste into the upstream folder. Vanilla kube does not define this API Group.

Reply:
Sorry!. Got it. Changed the defaults accordingly.